lib/puppet/pops/loader/task_instantiator.rb

```diff
@@ -60,7 +60,7 @@ def self.ensure_initialized
 
     @type_hash_t = tf.struct(
       tf.optional('description') => tf.string,
-      tf.optional('puppet_task_version') => tf.string,
+      tf.optional('puppet_task_version') => tf.integer,
```

Comment:
This should really have tests, but I understand wanting to get this merged asap. @thallgren can you file a ticket to add tests for all task metadata?